bin/initoverlayfs-install

```diff
@@ -17,7 +17,7 @@ exec_erofs() {
         "$SKIPCPIO_BIN" "/boot/initramfs-$kver.img" | "$CAT" | cpio -ivd
     popd
     rm -f "/boot/initoverlayfs-$kver.img"
-    mkfs.erofs "$erofs_compression" "/boot/initoverlayfs-$kver.img" ${INITRAMFS_DUMP_DIR}
```

Comment:
It may have been this change that was breaking the other PR, we should never have an empty "" string here, no quotes is what works here